### PR TITLE
blueos_startup_update: add 'remove_console' step

### PR DESCRIPTION
**Pull request by Williangalvani**
this install step was not in blueos_startup_update
https://github.com/bluerobotics/BlueOS/blob/master/install/boards/bcm_27xx.sh#L83-L84

### CODE REVIEW

File: core/tools/blueos_startup_update/blueos_startup_update.py [hunk 1]
```diff
@@ -446,6 +446,24 @@ def run_command_is_working():
...
 
 
+def remove_console():
+    """
+    removes "console=serial0,115200" from cmdline.txt
+    """
+    if not cmdline_file:
+        logger.error("cmdline unknown, aborting console patch")
+        return False
+    cmdline_content = load_file(cmdline_file).strip().split(" ")
```
Comment (joaoantoniocardoso) [inline core/tools/blueos_startup_update/blueos_startup_update.py:456]: why are we parsing the file differently here? Can we extract the correct way to parse  this file into a function?

Comment (Williangalvani) [reply core/tools/blueos_startup_update/blueos_startup_update.py:456]: yes, I intend to do that in another pr, where I also intend to stop opening it multiple times.
plus this is virtually the same structure in update_cgroups()

Comment (joaoantoniocardoso) [reply core/tools/blueos_startup_update/blueos_startup_update.py:456]: okay

**Review by joaoantoniocardoso — APPROVED** [review]

**Comment by patrickelectric** [comment]
Just saying that this step makes not possible to debug kernel problems via serial

**Pull request merged**